### ISSUE TITLE
app keeps refreshing infinitely when server is in maintenance mode

### ISSUE THREAD

**Issue by drs-project** [stefan-niedermann/nextcloud-deck#960]
When the nextcloud server is in maintenance mode the app shows this in the top bar, but keeps reloading the content.
This results in screen flickering an unusable app.

**Steps to reproduce the behavior:**
1. Put server in maintenance mode
2. Open deck app

**Expected behavior**
- App show that server is in maintenance mode (which is already great)
- Don't sync now
- Sync later (would be great)

**Screenshots**
![Screenshot_20210501-095036](https://user-images.githubusercontent.com/42349885/116775396-2c7ff680-aa63-11eb-8184-7c144b637db6.png)

**Versions**
 - Nextcloud: 20.0.9
 - Nextcloud Deck: 1.2.7
 - Nextcloud Android: 
 - Nextcloud Android Deck: 1.18.2

Besides that, thanks a lot for the great work on this really nice app!
Hope we can start using it team-wide soon...

**Comment by stefan-niedermann** [comment]
Thanks for the report! I'll have a look asap. What you see is not the refreshing spinner (compare to pull down 2 refresh which is the symbol for synchronization), but it is the "load from database" spinner.
Of course it is supposed to display the cards in this case - they are synchronized and locally available, so we should be able to display them :slightly_smiling_face: (And i am pretty sure that this worked once...)

**Comment by stefan-niedermann** [comment]
Some good news @drs-project - we found the root cause of the issue and (hopefully) resolved it.

Would be nice if you could confirm that the bug is fixed by testing [this `.apk` file](https://github.com/stefan-niedermann/nextcloud-deck/suites/2789297845/artifacts/61917085). You can install it safely parallel to your already installed Deck Android app.

**Comment by stefan-niedermann** [comment]
^ ping @drs-project

**Comment by drs-project** [comment]
the link to the apk doesnt seem to be active any more...

sorry, I couldnt make it earlier...

Am 21. Mai 2021 21:46:59 MESZ schrieb Niedermann IT-Dienstleistungen ***@***.***>:

**Comment by stefan-niedermann** [comment]
@drs-project yes it is, though you must be logged in in GitHub to be able to download it - i see you responded via E-Mail, so just open the issue in your browser, log in and then you should be able to download it 😉

**Comment by drs-project** [comment]
ok, now I made it :-)

I can confirm that behaviour is as expected, when server is in maintenance mode (not refreshing any more). A newly added card is synced when the server comes back online. - Great!

Additionally I checked what happens if a card is open on the handset while it is changed in the web interface. The changes from the web interface are synced with the next update (e.g. pull down). - Great!
